dart/lib/flat_buffers.dart

```diff
@@ -1241,3 +1246,72 @@ class _VTable {
     }
   }
 }
+
+// Allocator interface. This is flatbuffers-specific - only for [Builder].
+abstract class Allocator {
+  ByteData allocate(int size);
+
+  void deallocate(ByteData data);
```

Comment:
doc.

Also, this may want to go on the buffer class rather than here - unless it needs to access some private method on theinterface.

Reply:
Added the docs for now. I'll check about having a custom buffer.